detox/src/android/expect.js

```diff
@@ -115,14 +116,22 @@ class SwipeAction extends Action {
   }
 }
 
+class TakeElementScreenshot extends Action {
+  constructor() {
+    super();
+    this._call = invoke.callDirectly(DetoxActionApi.takeViewScreenshot());
+  }
+}
+
 class Interaction {
   constructor(invocationManager) {
     this._call = undefined;
     this._invocationManager = invocationManager;
   }
 
   async execute() {
-    await this._invocationManager.execute(this._call);
+    const resultObj = await this._invocationManager.execute(this._call);
+    return resultObj ? resultObj.result : undefined;
```

Comment:
This enables effectively using the result returned from invoke action on the device